### DISCUSSION TITLE
Sunshine is working in the background

### DISCUSSION THREAD

**Discussion by nikolanii** [LizardByte/community#93] [Sunshine] [answered]
Hello!

Is there a way to stop Sunshine from working in the background? Even after I turned it off.

**Comment by ReenigneArcher** [comment, marked as answer]
Disable the service

**Reply by nikolanii** [reply]
Thank you :)

I clicked uninstall service, but what does it do? I mean are there any cons to doing that?

**Reply by ReenigneArcher** [reply]
The service is what makes it run in the background, so disabling it only disables running in the background.
